azure-pipelines.yml

```
@@ -109,6 +111,16 @@ stages:
           appium-doctor
           echo "Appium doctor installed and executed"
 
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)